### PR TITLE
Moved the admin award year select to be closer to the search

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -690,6 +690,12 @@ table .ellipsis {
     overflow: hidden;
   }
 
+  .form_answers-index-page & {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Selector `form_answers-index-page` should be written in lowercase with hyphens

File: app/assets/stylesheets/admin/page-applications.scss [hunk 2]
```diff
@@ -416,6 +416,14 @@
   .control-label {
     display: block;
   }
+
+  .select2-container {
+    width: 300px !important;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: !important should not be used

File: app/assets/stylesheets/admin/page-applications.scss [hunk 3]
```diff
@@ -416,6 +416,14 @@
...
+
+    @media (max-width: 767px) {
+      width: 100% !important;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: !important should not be used

File: app/assets/stylesheets/admin/base.scss (hunk 1 above)
Comment (hammerfunk) [inline app/assets/stylesheets/admin/base.scss]: Ignore, the underscore came from controller name

**Comment by dreamfall** [comment]
looks good, let's wait for travis

**Pull request merged**